packages/plugin-e2e/tests/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { Page } from '@playwright/test';
+
+export const clickRadioButton = async (page: Page, label: string, options?: { exact?: boolean }) => {
+  try {
+    await page.getByLabel(label, options).click({ timeout: 1000 });
+  } catch (_) {
+    await page.getByText(label, options).click();
+  }
```

Comment:
It's not ideal to have to do this to retrieve a locator. However, had this been a _real_ e2e test in the clock panel plugin or in the sheets data source, I would have used data-testid instead to retrieve the locator and then this would not have been a problem.